R/query-string.R

```diff
@@ -8,29 +8,36 @@ queryStringFilter <- function(req){
 #' @importFrom utils URLdecode
 #' @noRd
 parseQS <- function(qs){
-  if (is.null(qs) || length(qs) == 0 || qs == ""){
+  # Is there data in the request?
+  if (is.null(qs) || length(qs) == 0 || qs == "") {
     return(list())
   }
-  if (stri_startswith_fixed(qs, "?")){
-    qs <- substr(qs, 2, nchar(qs))
-  }
-
-  qs <- gsub("+", " ", qs, fixed=TRUE)
+ # Is it JSON data?
+  if (stri_startswith_fixed(qs, "{")) {
```

Comment:
This wouldn't handle having any leading whitespace in front of the query string. I'm not sure if that's common or expected, but would it be worthwhile to trim whitespace from either end above this?